sites-available/README.md

```diff
@@ -9,4 +9,9 @@ example.com (handles traffic from both www.example.com and example.com)
 foobar.com (as above)
 test.foobar.com (handles traffic from both www.test.foobar.com and test.foobar.com)
 ```
+The HTTPS no-default server config requires a valid certificate (self signed will suffice).
+One can generate the certificate with: 
+```
+sudo openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /etc/nginx/ssl/nginx.key -out /etc/nginx/ssl/nginx.crt
```

Comment:
@AD7six Thanks for the reply and hints!

I did manage to solve the issue in adding the default certificate directives in the `http` block.

In my current configuration, multiple server blocks (99% percent of them https) I got to the state where the server was not serving any page without the default certificates in the default server block for https.

Now my config is working with this patch in the `nginx.conf -> http` block. I do not totally understand the reason why this works like this .. Do you have a link or simple explanation about this behaviour?